### DISCUSSION TITLE
Sidebar and Gradient not loading

### DISCUSSION THREAD

**Discussion by boojew** [matt8707/hass-config#169] [General]
I believe I have followed all the instructions, but I notice that both the sidebar and the gradient don't load. My CSS is terrible, so I am unable to see why..

<img width="1130" alt="image" src="https://user-images.githubusercontent.com/1015566/226366934-37d335db-866f-40b1-bb5a-98ef1f462355.png">

**Comment by boojew** [comment]
Ok - I found the issue w/ the gradient - I needed to copy over the www directory... but I still dont have the sidebar w/ time.

**Comment by MJ0230** [comment]
Did you find a solution for the missing sidebar. I am not able to get the sidebar to work as well

**Reply by cvonk** [reply]
Try adding `template: !include sidebar.yaml` to `configuration.yaml`

**Reply by cvonk** [reply]
The `unique_id` specified in `sidebar.yaml` caused havoc on my system.  In my case, it would show up in `Dev Tools > State`, but with a different identifier, such as `sensor.template_sensor`. 

Instead of using `unique_id`, I specified the `name`
```
- sensor:
    - name: sidebar
      state: template
```

After a reboot, it should now show up in `Dev Tools > States`.

Note that you may have to rename the sensor using the UI, as discussed in [[HA keeps changing my template sensor ids, adding suffixes](https://community.home-assistant.io/t/ha-keeps-changing-my-template-sensor-ids-adding-suffixes/601034)](https://community.home-assistant.io/t/ha-keeps-changing-my-template-sensor-ids-adding-suffixes/601034/4). 

I learned about the downside of `unique_id`, but at least it is working now:
<img width="530" height="625" alt="Screenshot 2025-11-07 003406" src="https://github.com/user-attachments/assets/0bc5c4bf-7282-44a8-8c6e-0bf0b063a2ae" />
